packages/nextra-theme/src/index.tsx

```diff
@@ -30,10 +28,118 @@ import {
 } from '@/components'
 import { DocumentContext, MDXLayoutNav, MDXLayoutOutline, MDXLayoutPagination, NavContext } from '@/layout'
 
+const MDXWrapper: NextraMDXContent = ({ children, toc }) => {
+  const { activePath } = useContext(NavContext)!
+  const { frontMatter, timestamp, readingTime } = useContext(DocumentContext)!
+  const { locale } = useRouter()
+  const lastUpdated = timestamp ? new Date(timestamp) : null
+
+  return (
+    <>
+      <div sx={{ pt: [null, null, null, Spacing['24px']] }}>
+        <div sx={{ display: [null, null, null, 'none'], mb: Spacing['32px'] }}>
+          <MDXLayoutNav mobile />
+        </div>
+
+        <article className="graph-docs-content" sx={mdxStyles}>
+          {activePath.length > 1 && (
+            <div className="graph-docs-current-group" sx={{ display: 'none' }}>
+              {activePath.map((item) => item.title).join(' > ')}
+            </div>
+          )}
+          {frontMatter.title ? <Heading.H1>{frontMatter.title}</Heading.H1> : null}
+          {(lastUpdated || readingTime) && (
+            <Paragraph size="14px">
+              {lastUpdated && (
+                <span>
+                  <span sx={{ color: 'White64' }}>Last updated:</span>{' '}
+                  <time
+                    dateTime={lastUpdated.toISOString()}
+                    // Removes hydration errors because `toLocaleDateString` show different results
+                    // in Node.js and in browser for some languages like `ar`
+                    suppressHydrationWarning
+                  >
+                    {lastUpdated.toLocaleDateString(locale, {
+                      month: 'long',
+                      day: 'numeric',
+                      year: 'numeric',
+                    })}
+                  </time>
+                </span>
+              )}
+              <span
+                sx={{
+                  mx: Spacing['16px'],
+                  display: 'inline-block',
+                  verticalAlign: 'top',
+                  width: '1px',
+                  height: '20px',
+                  bg: 'White16',
+                  '&:not(span + span), &:not(:has(+ span))': {
+                    display: 'none',
+                  },
+                }}
+              />
+              {!!readingTime?.minutes && (
```

Comment:
```suggestion
              {Boolean(readingTime?.minutes) && (
```
😇

Reply:
will rollback, because we have type error now https://github.com/graphprotocol/docs/actions/runs/10160360767/job/28096575978?pr=504

typescript works differently with `Boolean(...)` and `!!(...)` conversions

Reply:
Ah, interesting. Gotcha.